CHANGELOG.md

```diff
@@ -35,7 +41,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 
 [Unreleased]: https://github.com/BioJulia/BioServices.jl/compare/v0.1.3...HEAD
```

Comment:
@bicycle1885 This line should be changed to: `[Unreleased]: https://github.com/BioJulia/BioServices.jl/compare/v0.2.0...HEAD`. Otherwise people will be redirected to the wrong commits. Can you add this and retag the release?

Reply:
Oops, sorry. Will do now.